book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,211 @@
+               /                     \
+
+              Hash          Previous Bank-Diff-Merkle
+
+       /              \
+      /                \
+
+Hash(Account1)    Hash(Account2)
+
+```
+
+A state receipt contains only the updated state changes for the block.  A direct
+Merkle path to the current Bank-Merkle guarantees the state value at that bank
+hash, but it cannot be used to generate a “current” receipt to the latest state
+if the state modification occurred in some previous block.  There is no
+guarantee that the path provided by the validator is the latest one available
+out of all the previous Bank-Merkles.
+
+Clients that want to query the chain for a receipt of the "latest" state would
+need to create a transaction that would update the Merkle Path for that account,
+such as a credit of 0 lamports.
+
+###  Validator Votes
+
+Leaders should coalesce the validator votes by stake weight into a single entry.
+This will reduce the number of entries necessary to create a receipt.
+
+###  Chain of Entries
+
+A receipt is a PoH link from the payment or state Merkle Path to a consecutive
+list of validation votes.
+
+
+It contains the following:
+* State -> Bank-Merkle
+or
+* Transaction -> Entry-Merkle -> Block-Merkle -> Bank-Merkle
+
+And a vector of PoH entries:
+
+* Validator vote entries
+* Ticks
+* Light entries
+
+
+```
+/// This Entry definition skips over the transactions and only contains the
+/// hash of the transactions used to modify PoH.
+LightEntry {
+    /// The number of hashes since the previous Entry ID.
+    pub num_hashes: u64,
+    /// The SHA-256 hash `num_hashes` after the previous Entry ID.
+    hash: Hash,
+    /// The Merkle Root of the transactions encoded into the Entry.
+    entry_hash: Hash,
+}
+```
+
+The compressed entries are reconstructed from Entries and simply show the entry
+Merkle Root that was used to modify PoH, instead of the full transaction set.
+
+Clients do not need the starting vote state. The [fork
+selection](book/src/fork-selection.md) algorithm is defined such that only votes
+that appear after the transaction provide finality for the transaction, and
+finality is independent of the starting state.
+
+###  Verification
+
+A light client that is aware of the supermajority set validators can verify a
+receipt by following the Merkle Path to the PoH chain.  The Bank-Merkle is the
+Merkle Root and will appear in votes included in an Entry.  The light client can
+simulate [fork selection](book/src/fork-selection.md) for the consecutive votes
+and verify that the receipt is confirmed at the desired lockout threshold.
+
+### Synthetic State
+
+Synthetic state should be computed into the Bank-Merkle along with the bank
+generated state.
+
+For example:
+
+* Epoch validator accounts and their stakes and weights.
+* Computed fee rates
+
+Should have a path into the Bank-Merkle.  These values should live under known
+accounts, and therefore have an exact address in the Merkle Path.
```

Comment:
@aeyakovenko, what is meant by "an exact address in the Merkle Path" ?

Reply:
Unforgeable path.  Like the leader schedule, the path to that structure should be well known such that the reader can’t be fooled by an account value.  Basically, these things should have some structure that indicates the meaning of the value that is enforced by the bank

Reply:
So something like, "X is always the Nth leaf in the tree" ?

Reply:
@t-nelson I am not sure how other chains do this.  I was thinking we would type them.  so the leader schedule would be under some constant like:
`LEADER_SCHEDULE_MERKLE_ADDR=1`

Reply:
probably a cleaner thing would be 

```
Enum Roots {
  LeaderSchedule(Hash),
  EpochData(Hash),
  AccountData(Hash),
}
```

and the tree to the Bank root is computed from those intermediate roots.

Reply:
Right. Gotcha. That covers the OP.  Thanks!

RE other chains.  I'm not sure I've seen any trying to cover other state with a Merkle Root for SPV.  Usually just anchoring the TX in the block header.  Another point of research!